src/providers/mbed_provider/asym_encryption.rs

```diff
@@ -0,0 +1,96 @@
+// Copyright 2020 Contributors to the Parsec project.
+// SPDX-License-Identifier: Apache-2.0
+use super::{key_management, MbedProvider};
+use crate::authenticators::ApplicationName;
+use crate::key_info_managers::KeyTriple;
+use log::info;
+use parsec_interface::operations::{psa_asymmetric_decrypt, psa_asymmetric_encrypt};
+use parsec_interface::requests::{ProviderID, ResponseStatus, Result};
+use psa_crypto::operations::asym_encryption;
+use psa_crypto::types::key;
+use zeroize::Zeroize;
+
+impl MbedProvider {
+    pub(super) fn psa_asymmetric_encrypt_internal(
+        &self,
+        app_name: ApplicationName,
+        op: psa_asymmetric_encrypt::Operation,
+    ) -> Result<psa_asymmetric_encrypt::Result> {
+        info!("Mbed Provider - Asym Encrypt");
```

Comment:
I think those `info!` are not needed now that we have the `ingress`/`egress` trace everywhere

Reply:
Should they be removed from the other operations too?

Reply:
If you don't mind 👀